src/functional/ffront/decorator.py

```diff
@@ -471,7 +471,14 @@ def with_backend(self, backend: FencilExecutor) -> "FieldOperator":
         )
 
     def __gt_itir__(self) -> itir.FunctionDefinition:
-        return typing.cast(itir.FunctionDefinition, FieldOperatorLowering.apply(self.foast_node))
+        if hasattr(self, "__cached_itir"):
+            return getattr(self, "__cached_itir")  # noqa: B009
```

Comment:
What's the reason for the noqa instead of resolving the complaint?

Reply:
I can not tell you why, but `self.__cached_itir` fails. If you execute it in a debugger prompt it works. I have no clue why... I can leave a comment.

Reply:
ok